### PR TITLE
Fix etcd data dir

### PR DESCRIPTION
**Pull request by jgsqware**
Related to https://github.com/giantswarm/k8scloudconfig/pull/249

It add a dropin to execute the etcd data mount before setting ownership 

Closes https://github.com/giantswarm/k8scloudconfig/issues/228

### CODE REVIEW

**Comment by calvix** [comment]
by this you will just replace the content of the file, this is not systemd unit drop-in.

File: service/cloudconfig/v_0_1_0_master_template.go [hunk 1]
```diff
@@ -137,6 +137,12 @@ func (e *v_0_1_0MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
 			Owner:        FileOwner,
 			Permissions:  FilePermission,
 		},
+		{
```
Comment (calvix) [inline service/cloudconfig/v_0_1_0_master_template.go]: Also please put there comment `etcd_data_dir drop-in` so we have logical division :)

**Comment by jgsqware** [comment]
Hum, my bad, didn't push my last commit with this update. Copy paste problem

**Comment by jgsqware** [comment]
Tested and work well

**Pull request merged**